From de1b3fcfe10db99467d4029e821d92bab6e443c0 Mon Sep 17 00:00:00 2001
From: Ariadne Conill <ariadne@ariadne.space>
Date: Fri, 30 Aug 2024 16:09:31 -0700
Subject: [PATCH] s390/pci: follow alloc_pages in dma_map_ops name change

In commit 983e6b2636f0099dbac1874c9e885bbe1cf2df05, alloc_pages was renamed to
alloc_pages_op, but this was not changed for the s390 PCI implementation, most
likely due to upstream changes in the s390 PCI implementation which moved it
to using the generic IOMMU implementation after Linux 6.6 was released.

Signed-off-by: Ariadne Conill <ariadne@ariadne.space>
---
 arch/s390/pci/pci_dma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/s390/pci/pci_dma.c b/arch/s390/pci/pci_dma.c
index 99209085c75b..ce0f2990cb04 100644
--- a/arch/s390/pci/pci_dma.c
+++ b/arch/s390/pci/pci_dma.c
@@ -721,7 +721,7 @@ const struct dma_map_ops s390_pci_dma_ops = {
 	.unmap_page	= s390_dma_unmap_pages,
 	.mmap		= dma_common_mmap,
 	.get_sgtable	= dma_common_get_sgtable,
-	.alloc_pages	= dma_common_alloc_pages,
+	.alloc_pages_op	= dma_common_alloc_pages,
 	.free_pages	= dma_common_free_pages,
 	/* dma_supported is unconditionally true without a callback */
 };
-- 
2.46.0

